lib/scientist/experiment.rb

```diff
@@ -164,13 +174,9 @@ def name
     "experiment"
   end
 
-  # Internal: compare two observations, using the configured compare block if present.
+  # Internal: compare two observations, using the configured compare and error_compare lambdas if present.
```

Comment:
👍 to replacing "block", although you've replaced it here with "lambda" and elsewhere "proc"